stdlib/math.jou

```diff
@@ -0,0 +1,41 @@
+declare abs(x: int) -> int
+declare llabs(x: long) -> long
+declare fabs(x: double) -> double
+
+declare ceil(x: double) -> double
+declare floor(x: double) -> double
+declare round(x: double) -> double
+
+declare acos(x: double) -> double
+declare asin(x: double) -> double
+declare atan(x: double) -> double
+declare atan2(x: double, y: double) -> double
```

Comment:
The `atan2` function returns the angle of a point, given its x and y coordinates. It expects the y coordinate first, so let's name the arguments accordingly.
```suggestion
declare atan2(y: double, x: double) -> double
```